lib/gateway/Shard.js

```diff
@@ -843,6 +843,35 @@ class Shard extends EventEmitter {
                 }
                 break;
             }
+            case "VOICE_CHANNEL_EFFECT_SEND": {
+                const channel = this.client.getChannel(packet.d.channel_id) ?? {id: packet.d.channel_id};
+                const guild = this.client.guilds.get(packet.d.guild_id) ?? {id: packet.d.guild_id};
+                const user = this.client.users.get(packet.d.user_id) ?? {id: packet.d.user_id};
+                /**
+                 * Fired when a user send a voice channel effect
+                 * @event Client#voiceChannelEffectSend
+                 * @prop {Object} effect The effect that was sent
+                 * @prop {Guild | Object} effect.guild The guild. If the guild is not cached, this will be an object with an `id` key. No other property is guaranteed
+                 * @prop {VoiceChannel | Object} effect.channel The voice channel the effect was sent in. If the channel is not cached, this will be an object with an `id` key. No other property is guaranteed
+                 * @prop {User | Object} effect.user The user. If the user is not cached, this will be an object with an `id` key. No other property is guaranteed
```

Comment:
```suggestion
                 * @prop {Guild | Object} effect.guild The guild the effect was sent in. If the guild is not cached, this will be an object with an `id` key. No other property is guaranteed
                 * @prop {VoiceChannel | Object} effect.channel The voice channel the effect was sent in. If the channel is not cached, this will be an object with an `id` key. No other property is guaranteed
                 * @prop {User | Object} effect.user The user that sent the effect. If the user is not cached, this will be an object with an `id` key. No other property is guaranteed
```